tutorials/caffe_prototxt_usage_1.md

```diff
@@ -34,4 +34,13 @@ In order to export a Caffe Model from Fabrik:
   caffe train \
     -gpu 0 \
     -solver solver.prototxt
-  ```
\ No newline at end of file
+  ```
+
+### Code template
+[caffe_sample.py](../example/caffe/code_samples/caffe_sample.py)
+The code sample loads the model specified by the user and trains the model.
```

Comment:
Change this to `The code sample in [caffe_sample.py](../example/caffe/code_samples/caffe_sample.py) loads`